Bottom: c306df7bf7a7ca3c143c0093a82565c6943a3620
Top:    cd33ccc503dec8295ae3bfc043527a8a7bbf1c5f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 14:11:15 +0100

Refresh of demo-of-exporter-specific-expo

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 5b24567..2e7e34f 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -500,7 +500,6 @@ common_export_region (HID *hid, BoxType *region)
   hid->set_color (Output.pmGC, "erase");
   hid->set_color (Output.bgGC, "drill");
 
-
   PCB->Data->SILKLAYER.Color = PCB->ElementColor;
   PCB->Data->BACKSILKLAYER.Color = PCB->InvisibleObjectsColor;
